src/buttonGroup/style.ts

```diff
@@ -0,0 +1,45 @@
+import css from 'styled-jsx/css'
+
+import { space } from '_utils/branding'
+
+export default css`
+  .kirk-button-group {
+    display: flex;
+  }
+
+  .kirk-button-group.kirk-button-group-column {
+    flex-direction: column;
+  }
+  .kirk-button-group.kirk-button-group-column :global(.kirk-button) {
+    justify-content: center;
+  }
+  .kirk-button-group.kirk-button-group-column > :global(.kirk-button + .kirk-button) {
+    margin-top: ${space.m};
+  }
+
+  .kirk-button-group.kirk-button-group-column.kirk-button-group.kirk-button-group-reverse {
```

Comment:
is the doubling of `.kirk-button-group` necessary?

Reply:
Oups! My bad ^^